### DISCUSSION TITLE
How do I [[]] this for internal link in tables ?

### DISCUSSION THREAD

**Discussion by arromattic** [TriliumNext/Trilium#4071] [Q&A] [answered]
###  I am trying to create wikipedia's navigation box like this using table in trilium
![image](https://github.com/zadam/trilium/assets/133383364/1965fa67-f2a4-4e31-9e30-5d7d50884bc6)
### But when i am using [[]] this to link pages it's not working
what shortcut should i use or is there ?

**Comment by XXXJetfireXXX** [comment, marked as answer]
Internal links are made by selecting text and pressing `Ctrl+L` or by choosing this option in the popup menu:
![image](https://github.com/zadam/trilium/assets/43646178/374d1f45-87e6-4440-9f45-056faaaa44be)

**Reply by arromattic** [reply]
Thanks `Ctrl+L` works for me
